checks/raw/security_policy.go

```diff
@@ -199,7 +199,7 @@ func collectPolicyHits(policyContent []byte) []checker.SecurityPolicyInformation
 	// pattern for URLs
 	reURL := regexp.MustCompile(`(http|https)://[a-zA-Z0-9./?=_%:-]*`)
 	// pattern for emails
-	reEML := regexp.MustCompile(`\b[A-Za-z0-9._%+-]+@[A-Za-z0-9.-]+\.[A-Za-z]{2,6}\b`)
+	reEML := regexp.MustCompile(`\b[A-Za-z0-9._%+-]+(@|\\?\[at\\?\])[A-Za-z0-9.-]+\.[A-Za-z]{2,6}\b`)
```

Comment:
do people always do lowercase `[at]`?

Reply:
I haven't seen that yet, but I suppose its possible.

Reply:
we can always add it later if other forms are seen